### ISSUE TITLE
nushell: Deprecated syntax for init.nu

### ISSUE THREAD

**Issue by xqtc161** [rsteube/carapace-bin#2174]
### Current Behavior

When using tab-completion I get the following error:

```
 Error:   × Automatically spreading lists is deprecated
   ╭─[/Users/xqtc/.cache/carapace/init.nu:18:1]
18 │
19 │   carapace $spans.0 nushell $spans                                                                                                                    
                         ───┬──                                                                                                                                                                                      
                            ╰── Spreading lists automatically when calling external commands is deprecated and will be removed in 0.91.                                                                                                                                        20 │   | from json
   ╰────
```

A potential fix could be prepending `...` in front of `$spans` since not supplying `...` is deprecated since nushell 0.89, which I'm using rn.

### Expected Behavior

_No response_

### Steps To Reproduce

1. Nushell with carapace (+starship and atuin but that shouldn't matter)
2. Hit tab completion after installing carapace-bin according to the docs.

### Version

0.29.0

### OS

- [X] Linux
- [X] OSX
- [ ] Windows

### Shell

- [ ] Bash
- [ ] Elvish
- [ ] Fish
- [X] Nushell
- [ ] Oil
- [ ] Powershell
- [ ] Xonsh
- [ ] Zsh

### Anything else?

_No response_

**Comment by rsteube** [comment]
Thanks for noticing me. This was recently fixed with https://github.com/rsteube/carapace-bin/pull/2166/files .
That breaks compability with the old version though :smile: .
[Packaging](https://repology.org/project/nushell/versions) looks fine so it should be ok to tag a release soon.

For now you can hardcode it in your `config.nu`:
```nushell
let external_completer = {|spans|
    carapace $spans.0 nushell ...$spans | from json
}

...

    completions: {
        external: {
            completer: $external_completer # check 'carapace_completer' above as an example
        }
    }
```

**Comment by rsteube** [comment]
Fixed in v0.29.1
